homeassistant/components/husqvarna_automower/sensor.py

```diff
@@ -26,6 +26,165 @@
+    "invalid_sub_device_combination",
+    "invalid_system_configuration",
+    "left_brush_motor_overloaded",
+    "lift_sensor_defect",
+    "lifted",
+    "limited_cutting_height_range",
+    "limited_cutting_height_range",
+    "loop_sensor_defect",
+    "loop_sensor_problem_front",
+    "loop_sensor_problem_left",
+    "loop_sensor_problem_rear",
+    "loop_sensor_problem_right",
+    "low_battery",
+    "memory_circuit_problem",
+    "mower_lifted",
+    "mower_tilted",
+    "no_accurate_position_from_satellites",
+    "no_confirmed_position",
+    "no_drive",
+    "no_loop_signal",
+    "no_power_in_charging_station",
+    "no_response_from_charger",
+    "outside_working_area",
+    "poor_signal_quality",
+    "reference_station_communication_problem",
+    "right_brush_motor_overloaded",
+    "safety_function_faulty",
+    "settings_restored",
+    "sim_card_locked",
+    "sim_card_locked",
+    "sim_card_locked",
+    "sim_card_locked",
+    "sim_card_not_found",
+    "sim_card_requires_pin",
+    "slipped_mower_has_slipped_situation_not_solved_with_moving_pattern",
+    "slope_too_steep",
+    "sms_could_not_be_sent",
+    "stop_button_problem",
+    "stuck_in_charging_station",
+    "switch_cord_problem",
+    "temporary_battery_problem",
+    "temporary_battery_problem",
+    "temporary_battery_problem",
+    "temporary_battery_problem",
+    "temporary_battery_problem",
+    "temporary_battery_problem",
+    "temporary_battery_problem",
+    "temporary_battery_problem",
+    "tilt_sensor_problem",
+    "too_high_discharge_current",
+    "too_high_internal_current",
+    "trapped",
+    "ultrasonic_problem",
+    "ultrasonic_sensor_1_defect",
+    "ultrasonic_sensor_2_defect",
+    "ultrasonic_sensor_3_defect",
+    "ultrasonic_sensor_4_defect",
+    "unexpected_cutting_height_adj",
+    "unexpected_error",
+    "upside_down",
+    "weak_gps_signal",
+    "wheel_drive_problem_left",
+    "wheel_drive_problem_rear_left",
+    "wheel_drive_problem_rear_right",
+    "wheel_drive_problem_right",
+    "wheel_motor_blocked_left",
+    "wheel_motor_blocked_rear_left",
+    "wheel_motor_blocked_rear_right",
+    "wheel_motor_blocked_right",
+    "wheel_motor_overloaded_left",
+    "wheel_motor_overloaded_rear_left",
+    "wheel_motor_overloaded_rear_right",
+    "wheel_motor_overloaded_right",
+    "work_area_not_valid",
+    "wrong_loop_signal",
+    "wrong_pin_code",
+    "zone_generator_problem",
+]
+
+RESTRICTED_REASONS: list = [
+    RestrictedReasons.ALL_WORK_AREAS_COMPLETED.lower(),
+    RestrictedReasons.DAILY_LIMIT.lower(),
+    RestrictedReasons.EXTERNAL.lower(),
+    RestrictedReasons.FOTA.lower(),
+    RestrictedReasons.FROST.lower(),
+    RestrictedReasons.NONE.lower(),
+    RestrictedReasons.NOT_APPLICABLE.lower(),
+    RestrictedReasons.PARK_OVERRIDE.lower(),
+    RestrictedReasons.SENSOR.lower(),
+    RestrictedReasons.WEEK_SCHEDULE.lower(),
+]
```

Comment:
```suggestion
```

Reply:
I already had it like this, but I changed it. See comment [here](https://github.com/home-assistant/core/pull/113165#discussion_r1545754481). Should I change it back?